src/mocks/entity/mockEntity.ts

```diff
@@ -44,3 +83,144 @@ export const mockFolderEntityHeader: EntityHeader = {
   createdBy: '', // TODO: Replace with a valid mock user ID
   modifiedBy: '',
 }
+
+export const mockProjectEntityBundle: EntityBundle = {
+  entity: {
+    name: 'A Mock Project',
+    id: MOCK_PROJECT_ID,
+    etag: '7849ff2c-1c93-4104-adcf-9e6d6b0c50b5',
+    createdOn: '2020-11-18T20:05:06.519Z',
+    modifiedOn: '2021-05-19T12:44:35.457Z',
+    createdBy: `${MOCK_USER_ID}`,
```

Comment:
In general, I'm trying to associate all of our mocks so when a component loads multiple related objects, all data ends up being valid.